kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ReplaceQuestions.vue

```diff
@@ -251,9 +251,11 @@
       }
 
       function toggleInReplacements(question) {
-        const replacementIds = replacements.value.map(q => q.id);
-        if (replacementIds.includes(question.item)) {
-          replacements.value = replacements.value.filter(q => q.item !== question.item);
+        const replacementIds = replacements.value.map(r => r.question_id);
```

Comment:
I think this is an over correction. I suspect just changing id to item on the first line would be sufficient here.

The issue with the change made here is that question_id is not guaranteed to be unique across exercises, so the item property (which is a compound of the exercise_id and question_id) guarantees that the correct item is being replaced.

The original issue was due to the id property being removed from the question objects in favour of the previously and more widely used item property.